src/StorageSync/StorageSync/Interop/Clients/SyncServerRegistrationClientBase.cs

```diff
@@ -160,9 +199,11 @@ public void Dispose()
                 throw new ServerRegistrationException(ServerRegistrationErrorCode.RegisterOnlineSyncRegistrationFailed);
             }
 
-            // Setting ServerCertificate from request resource to response resource so that it can be used by Monitoring pipeline
-            resultantRegisteredServerResource.ServerCertificate = Convert.ToBase64String(serverRegistrationData.ServerCertificate);
-
+            //if (isCertificateRegistration)
```

Comment:
again seems like this shouldn't be commented out?

Reply:
not in v19